### PR TITLE
Hca fig4

### PR DESCRIPTION
**Pull request by Hugemiler**
PR for HCA on Figure 4 and Supplementary tables (last features from Mar 06-10 sprint before final tweaks)

### CODE REVIEW

**Comment by kescobo** [comment]
Please don't open PR and merge immediately - give me a chance to review. You could still continue working by branching off of this branch.

File: manuscript/TableS2.jl [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## 2. Calculating the Figures of Merit
+tableS2 = reduce(
+    vcat,
+    [ DataFrame(:variable => ordered_brain_segments_list[i], :Test_MAPE => mean(brain_models[ordered_brain_segments_list[i]].merits.Test_MAPE), :Test_Cor => mean(brain_models[ordered_brain_segments_list[i]].merits.Test_Cor)) for i in eachindex(ordered_brain_segments_list) ]
+)
+
+tableS2.variable = uppercasefirst.(map(x -> replace(x, "-"=>" "), tableS2.variable))
+
+CSV.write("TableS2.csv", tableS2)
```
Comment (kescobo) [inline manuscript/TableS2.jl:67]: In future, please use eg `tablefiles(TableS2.csv")`, or in this case maybe `"manuscript/assets/TableS2.csv"`

File: manuscript/TableS2.jl [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    "left-insula", "right-insula",
+    "cerebellar-vermal-lobules-I-V", "cerebellar-vermal-lobules-VI-VII", "cerebellar-vermal-lobules-VIII-X",
+    "Brain-stem", "CSF"
+]
+
+## 2. Calculating the Figures of Merit
+tableS2 = reduce(
+    vcat,
+    [ DataFrame(:variable => ordered_brain_segments_list[i], :Test_MAPE => mean(brain_models[ordered_brain_segments_list[i]].merits.Test_MAPE), :Test_Cor => mean(brain_models[ordered_brain_segments_list[i]].merits.Test_Cor)) for i in eachindex(ordered_brain_segments_list) ]
+)
```
Comment (kescobo) [inline manuscript/TableS2.jl:63]: Consider also `mapreduce`, eg 

```
tableS2 = mapreduce( vcat, eachindex(ordered_brain_segments_list)) do i
    DataFrame(:variable => ordered_brain_segments_list[i],
              :Test_MAPE => mean(brain_models[ordered_brain_segments_list[i]].merits.Test_MAPE), 
              :Test_Cor => mean(brain_models[ordered_brain_segments_list[i]].merits.Test_Cor))
end
```

I personally find that cleaner than the comprehension

File: manuscript/Figure4.md [hunk 3]
```diff
@@ -369,9 +385,9 @@ end
 CairoMakie.scatter!(axC, xs .+ rand(Normal(0, 0.1), length(xs)), ys;)
 Legend(C_subfig[1,2], [MarkerElement(; marker=:rect, color = ColorSchemes.Set3_12[i]) for i in 1:length(imp_bugs)], imp_bugs)
```
Comment (kescobo) [inline manuscript/Figure4.md:386]: I think we maybe want to reverse the colors or the legend. Right now it's `figure: left-> right` and `legend: bottom-> up`. I feel like left-> right, top-> down is more intuitive. Might be as simple as `... for i in eachindex(imp_bugs)], reverse(imp_bugs)...`

File: manuscript/TableS2.jl (hunk 2 above)
Comment (Hugemiler) [reply manuscript/TableS2.jl:63]: FIxed on #135

**Pull request merged**